### PR TITLE
festival.json : valentine update both en & th

### PR DESCRIPTION
**Pull request by Peachpiggies**

### CODE REVIEW

File: responses/festivals.json [hunk 1]
```diff
@@ -48,6 +48,23 @@
...
+                "Happy valentine! Do you have chocolate for me?"
+            ],
+            "nonfes": [
+                "Ah, it's not valentine's day, but wish you luck for love."
+            ]
+        },
+        "list_of_words": ["valentine"],
+        "is_single_response": false,
+        "required_word": ["valentine"],
+        "date": [14],
```
Comment (StrixzIV) [inline responses/festivals.json]: This data is an integer data so change this to `"date: 14"`

File: responses/festivals_th.json [hunk 2]
```diff
@@ -48,5 +48,22 @@
...
+                "สุขสันต์วันวาเลนไทน์นะคะ มีช็อกโกแลตให้หนูไหมคะ"
+            ],
+            "nonfes": [
+                "น่าเสียดายนะคะที่ไม่ใช่วันวาเลนไทน์ แต่ก็ขอให้สมหวังกับความรักนะคะ"
+            ]
+        },
+        "list_of_words": ["วาเลนไทน์"],
+        "is_single_response": false,
+        "required_word": ["วาเลนไทน์"],
+        "date": [14],
```
Comment (StrixzIV) [inline responses/festivals_th.json]: This data is an integer data so change this to `"date: 14"`

**Review by StrixzIV — CHANGES_REQUESTED** [review]
Fix the date pls

**Review by Peachpiggies — COMMENTED** [review]
This will do it now.

**Review by Peachpiggies — COMMENTED** [review]
This will do it right.
:3

**Review by StrixzIV — APPROVED** [review]

**Pull request merged**